### PR TITLE
[luci-micro] Add Fill kernel to PAL

### PR DESCRIPTION
**Pull request by SlavikMIPT**
This commit adds Fill kernel to PAL.

ONE-DCO-1.0-Signed-off-by: Vyacheslav Bazhenov <v.bazhenov@partner.samsung.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
Please split `luci-interpreter` and `luci-micro` files into separate PRs.

File: compiler/luci-interpreter/pal/mcu/PALFill.h [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/*
+* Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
```
Comment (binarman) [inline compiler/luci-interpreter/pal/mcu/PALFill.h:2]: Did you copy paste this?

```suggestion
* Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
```

Comment (binarman) [reply compiler/luci-interpreter/pal/mcu/PALFill.h:2]: I am also have concerns that we need this pal in "big" luci-interpreter

File: compiler/luci-micro/luci-interpreter/pal/linux/PALreference_ops.h [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef LUCI_INTERPRETER_PAL_REFERENCE_OPS_H
+#define LUCI_INTERPRETER_PAL_REFERENCE_OPS_H
+
+#include <tensorflow/lite/kernels/internal/reference/reference_ops.h>
+
+#endif // LUCI_INTERPRETER_PAL_REFERENCE_OPS_H
```
Comment (binarman) [inline compiler/luci-micro/luci-interpreter/pal/linux/PALreference_ops.h:22]: Why do we need both this header and `PALFill.h`?

**Review by BalyshevArtem — APPROVED** [review]
LGTM

**Review by binarman — APPROVED** [review]
LGTM

**Pull request merged**